bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -245,12 +231,66 @@ pub struct TracingFileConfiguration {
     max_files: Option<u64>,
 }
 
+#[derive(PartialEq, PartialOrd)]
+enum LogTarget {
+    Hyper,
+    MatrixSDKFFI,
+    MatrixSDKClient,
+    MatrixSDKCrypto,
+    MatrixSDKCryptoAccount,
+    MatrixSDKOIDC,
+    MatrixSDKHTTPClient,
+    MatrixSDKSlidingSync,
+    MatrixSDKBaseSlidingSync,
+    MatrixSDKUITimeline,
+    MatrixSDKEventCache,
+    MatrixSDKEventCacheStore,
+}
+
+impl LogTarget {
+    fn as_str(&self) -> &'static str {
+        match self {
+            LogTarget::Hyper => "hyper",
+            LogTarget::MatrixSDKFFI => "matrix_sdk_ffi",
+            LogTarget::MatrixSDKClient => "matrix_sdk::client",
+            LogTarget::MatrixSDKCrypto => "matrix_sdk_crypto",
+            LogTarget::MatrixSDKCryptoAccount => "matrix_sdk_crypto::olm::account",
+            LogTarget::MatrixSDKOIDC => "matrix_sdk::oidc",
+            LogTarget::MatrixSDKHTTPClient => "matrix_sdk::http_client",
+            LogTarget::MatrixSDKSlidingSync => "matrix_sdk::sliding_sync",
+            LogTarget::MatrixSDKBaseSlidingSync => "matrix_sdk_base::sliding_sync",
+            LogTarget::MatrixSDKUITimeline => "matrix_sdk_ui::timeline",
+            LogTarget::MatrixSDKEventCache => "matrix_sdk::event_cache",
+            LogTarget::MatrixSDKEventCacheStore => "matrix_sdk_sqlite::event_cache_store",
```

Comment:
For the event cache pack, we'd need also `LogTarget::MatrixSdkBaseEventCache`. But I can do that as a follow-up, if you'd like :)

Reply:
I've added`MatrixSdkBaseEventCache` that defaults to `Info` like the others and I've also added an `matrix_sdk` wide one that also defaults to info but won't drop lower than that.